src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    public abstract double calculateArea();
```

Comment:
```suggestion
    public String color;
```

create constructor and get method

Also, create AreaCalculator interface with double calculateArea() method